src/hooks/RouterHooks.js

```diff
@@ -8,44 +8,43 @@ import { useWorkspaceData } from '../state/hooks/WorkspaceHooks';
 import { ConfigUtils } from '../utils';
 import { useSortedScenarioList } from './ScenarioListHooks';
 
-export const useRedirectFromInstanceToScenarioView = () => {
+export const useRedirectToScenarioView = (view) => {
```

Comment:
IIUC, this hook only redirects to scenario view (or workspace view) when the targeted route is invalid?
Can we rename it to `useRedirectInvalidRoutes` or maybe `useRedirectIfViewDisabled`?

Reply:
Good idea ! :+1: I renamed it to useRedirectFromDisabledView :wink: